### DISCUSSION TITLE
Emergency "drift" for an Helm release

### DISCUSSION THREAD

**Discussion by mimmus** [fluxcd/flux2#3506] [Q&A]
Hi,
how can temporarily exclude an Helm release from Flux reconciliation process, updating it by hand (using "helm upgrade ..."), as an emergency measure to solve an issue with current release?

I cannot change now the release in the Git repo for reasons not important here.

Thanks

**Comment by jack-evans** [comment]
You could `flux suspend helmrelease` and then make your adjustments, once done, just resume and it'll be reconciled again to what the git state is
